public/app/features/plugins/PluginPage.tsx

```diff
@@ -379,7 +302,7 @@ class PluginPage extends PureComponent<Props, State> {
                 {plugin && (
                   <section className="page-sidebar-section">
                     {this.renderVersionInfo(plugin.meta)}
-                    {this.renderSidebarIncludes(plugin.meta.includes)}
+                    {isAdmin && this.renderSidebarIncludes(plugin.meta.includes)}
```

Comment:
This now does not show includes for non admins. Pages API routes need to work but some of the pages are configuration ones. In here they would all be shown. Non admin user can access available pages through App navigation (which will only show non config pages to him) so this seems sensible to me.